### PR TITLE
PR#263 Parser created for show snmp

### LINKED ISSUE #17: 6 - show snmp
```
device1#show snmp
Chassis: FWU6550A9EB
37511918 SNMP packets input
    0 Bad SNMP version errors
    637811 Unknown community name
    0 Illegal operation for community name supplied
    0 Encoding errors
    350597419 Number of requested variables
    2770 Number of altered variables
    908861 Get-request PDUs
    32841743 Get-next PDUs
    1527 Set-request PDUs
    0 Input queue packet drops (Maximum queue size 1000)
37613240 SNMP packets output
    0 Too big errors (Maximum packet size 1500)
    0 No such name errors
    0 Bad values errors
    0 General errors
    640 Response PDUs
    739133 Trap PDUs
Packets currently in SNMP process input queue: 0
SNMP global trap: enabled

SNMP logging: enabled
    Logging to 18.44.1.51.164, 0/10, 507795 sent, 231338 dropped.
```
```
device2#show snmp
Chassis: FCE2310A48M
Contact: "something@cisco.com"
Location: "To be filled in"
35072424 SNMP packets input
    0 Bad SNMP version errors
    2089 Unknown community name
    68 Illegal operation for community name supplied
    0 Encoding errors
    341796946 Number of requested variables
    7328 Number of altered variables
    954173 Get-request PDUs
    31072480 Get-next PDUs
    3878 Set-request PDUs
    0 Input queue packet drops (Maximum queue size 1000)
35598123 SNMP packets output
    0 Too big errors (Maximum packet size 1500)
    1 No such name errors
    0 Bad values errors
    0 General errors
    322899 Response PDUs
    527788 Trap PDUs
Packets currently in SNMP process input queue: 0
SNMP global trap: enabled

SNMP logging: enabled
    Logging to 13.43.1.12.151, 0/10, 102045 sent, 3065 dropped.
    Logging to 12.44.133.10.111, 0/10, 102045 sent, 3065 dropped.
    Logging to 116.75.154.186.169, 0/10, 201075 sent, 11383 dropped.
    Logging to 126.35.67.19.108, 0/10, 102045 sent, 3065 dropped.
```
```
device3#show snmp
Chassis: FCG2219A37C
Contact: example@cisco.com
Location: Bldg 12 Noc Row 3
124561583 SNMP packets input
    0 Bad SNMP version errors
    18238 Unknown community name
    176 Illegal operation for community name supplied
    0 Encoding errors
    527651116 Number of requested variables
    3418 Number of altered variables
    3997679 Get-request PDUs
    43981910 Get-next PDUs
    2191 Set-request PDUs
    0 Input queue packet drops (Maximum queue size 1000)
142422017 SNMP packets output
    0 Too big errors (Maximum packet size 1500)
    52 No such name errors
    0 Bad values errors
    0 General errors
    75930580 Response PDUs
    17878672 Trap PDUs
Packets currently in SNMP process input queue: 0
SNMP global trap: enabled

SNMP logging: enabled
    Logging to 12.22.145.10.162, 2/10, 4477846 sent, 5337 dropped.
    Logging to 171.71.168.146.102, 6/10, 8962049 sent, 26431 dropped.
    Logging to 11.41.1.52.166, 2/10, 4403187 sent, 3812 dropped.
```
```
device4#show snmp
Chassis: FOX1962GXCM
Contact: "address@cisco.com"
Location: "To be filled in"
2328857 SNMP packets input
    0 Bad SNMP version errors
    894360 Unknown community name
    11 Illegal operation for community name supplied
    0 Encoding errors
    8399832 Number of requested variables
    0 Number of altered variables
    395637 Get-request PDUs
    58155 Get-next PDUs
    11 Set-request PDUs
    0 Input queue packet drops (Maximum queue size 1000)
1443766 SNMP packets output
    0 Too big errors (Maximum packet size 1500)
    12 No such name errors
    0 Bad values errors
    0 General errors
    1306589 Response PDUs
    9274 Trap PDUs

SNMP logging: enabled
    Logging to 10.45.48.23.190, 0/10, 9079 sent, 195 dropped.
```
```
device5#show snmp
Chassis: FOX1263GM4M
Contact: address@cisco.com
Location: SJCT
17835406 SNMP packets input
    0 Bad SNMP version errors
    520586 Unknown community name
    1521123 Illegal operation for community name supplied
    0 Encoding errors
    236025384 Number of requested variables
    936 Number of altered variables
    8863583 Get-request PDUs
    3342039 Get-next PDUs
    468 Set-request PDUs
    0 Input queue packet drops (Maximum queue size 1000)
19986239 SNMP packets output
    4 Too big errors (Maximum packet size 1500)
    4389 No such name errors
    0 Bad values errors
    0 General errors
    16888105 Response PDUs
    2671419 Trap PDUs

SNMP logging: enabled
    Logging to 13.35.48.66.152, 0/10, 347698 sent, 186588 dropped.
    Logging to 13.35.48.76.162, 0/10, 347697 sent, 186587 dropped.
    Logging to 10.35.42.77.192, 0/10, 347696 sent, 186587 dropped.
    Logging to 171.59.43.10.162, 0/10, 347696 sent, 186587 dropped.
    Logging to 191.70.168.156.162, 0/10, 347694 sent, 186589 dropped.
```

### PR DESCRIPTION
**Pull request by qduk**
Let me know what you guys think for the dictionary key names (some seem long to me but I didn't want to lose clarity for the sake of shorter keys) and the schema. I'd definitely like to know how you guys would set the schema if it different than what I have.

### CODE REVIEW

File: src/genie/libs/parser/iosxe/show_snmp.py [hunk 1]
```diff
@@ -79,3 +80,325 @@ def cli(self, output=None):
...
+        }
+    }
+
+# ==========================
+# Parser for 'show snmp'
+# ==========================
+class ShowSnmp(ShowSnmpSchema):
+
+    ''' Parser for "show snmp" '''
+
```
Comment (itdependsnetworks) [inline src/genie/libs/parser/iosxe/show_snmp.py:142]: Don't have to on this one, as I don't see a big use case for comparing this, but consider using the exclude key as seen here: `https://github.com/CiscoTestAutomation/genieparser/blob/213a1f32f152b9e029864341cec64f8e7716ace3/src/genie/libs/parser/viptela/show_bfd_sessions.py#L45`

**Comment by itdependsnetworks** [comment]
Looks good, for long key names, Cisco seems to have thoughts, I'll let them come up with them. 

(don't forget to label cisco_pr once that is done)

File: src/genie/libs/parser/iosxe/show_snmp.py (hunk 1 above)
Comment (qduk) [reply src/genie/libs/parser/iosxe/show_snmp.py:142]: Added.

**Comment by qduk** [comment]
[#263](https://github.com/CiscoTestAutomation/genieparser/pull/263) created for Cisco

**Pull request merged**